.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: ruby
+rvm:
+  - 2.0
+  - 2.1
```

Comment:
It seems fine to me:
https://travis-ci.org/simon3z/kubeclient/builds/50911341

ruby 2.0.0p598 (2014-11-13 revision 48408) [x86_64-linux]